### PR TITLE
[Modular] The Cybersun Arsenal - Two New Traitor Items.

### PR DESCRIPTION
**Pull request by unit0016**
## About The Pull Request
Adds two new Traitor Items.
The first is a variant of the agent ID card. It costs 15TC, but it's wildcard is captain-level. It can copy any amount of station access as a result.

The second is more complicated. It's a variant of the chameleon headset that costs 25TC. For that incredible uptick in cost, you get:

Freerange. Your headset can be dialed to any frequency (minus syndiecomms, which has protection from freerange) and talk on it as if you had the proper encryption keys. Sadly you lose access to hearing common while using this feature.
Loudmode. Your headset can be LOUD. Good for hostage situations or other scenarios you want everyone's attention, but don't want to break into some head of staff's office just to get a megaphone.
Flashbang protection. Self explanatory, save your ears from all the shooting you'll hear as security runs for you!
image

## Why It's Good For The Game
More traitor items is always good. Expands the variety of what we can expect from antagonists and how they'll act, opens the gateways to be more creative.

## Changelog
:cl:
add: Cybersun has released upgraded stock to uplinks across charted and uncharted space.
/:cl:

### CODE REVIEW

**Comment by FlamingLily** [comment]
25 TC for a headset? does it AT LEAST come with an encryption key?

**Comment by unit0016** [comment]
> 25 TC for a headset? does it AT LEAST come with an encryption key?

If you use it properly, it comes with _all_ encryption keys. Minus Syndiecomms and Binary.

**Comment by FlamingLily** [comment]
> 
> 
> > 25 TC for a headset? does it AT LEAST come with an encryption key?
> 
> If you use it properly, it comes with _all_ encryption keys. (In function.) Minus Syndiecomms and Binary.

So it's **25** TC for a worse captain's headset.

Something you could achieve in like, 4, minimum? With the real captain's headset? Provided they don't take their spare headset, of course.

**Comment by unit0016** [comment]
> > > 25 TC for a headset? does it AT LEAST come with an encryption key?
> > 
> > 
> > If you use it properly, it comes with _all_ encryption keys. (In function.) Minus Syndiecomms and Binary.
> 
> So it's **25** TC for a worse captain's headset.
> 
> Something you could achieve in like, 4, minimum? With the real captain's headset? Provided they don't take their spare headset, of course.

Zero. Zero TC, actually. You don't need to even use a single traitor item to lay your hands on the captain's headset, just a fireaxe and a good eye for greytiding. Same could be said of the syndie jaws of life, of the syndie magboots, or mulligan, etc etc.

This is priced appropriately. If it was priced lower, everyone would buy it for loudmode and basically free security comms and flashbang protection. If it was priced higher, nobody would buy it because it's too expensive for it's niche.

This is a stealth tool. There's nothing stealthy about emagging your way into the captain's office to grab their headset, and it won't help you blend in if you're trying to slip into a department unnoticed without taking anyone down. 
Buying traitor items is about buying practical use for time. A tradeoff between the tools you have now, the money you have left, and the time left in the shift. You could absolutely amass an army of guns and hardsuits and be able to solo security without using a single telecrystal.
But the shuttle would be leaving before you even tried. A certain spacetider from /tg/station manuel might be able to confirm as such.
It isn't about what's objective. It's about what you need, what you want, and what you have to do. 
Objectively, syndiecomms are the best tools for traitors in the game, as it allows you to sync up with the comms agent and other traitors to work in tandem to ruin security's day. But in practice, next to nobody buys them. Maybe two people in a shift.

tl;dr, yes, and?

**Comment by FlamingLily** [comment]
25 TC is wayyyyyyy overpriced.

What does it bring. Loudmode? steal your head's headset. (Hell maybe even ask, you could probably bullshit your way into getting it, especially if you don't want the encryption key.) or a megaphone

'bang protection? Arrivals checkpoint. Departmental sec posts.

Chameleon? Cham kit.

Ability to speak on any comms channel? Yeah at the cost of not hearing common or said channel you plan to speak on w/o a syndie key anyway.

25 TC is absolutely overpriced for this.

**Comment by unit0016** [comment]
> 25 TC is wayyyyyyy overpriced.
> 
> What does it bring. Loudmode? steal your head's headset. (Hell maybe even ask, you could probably bullshit your way into getting it, especially if you don't want the encryption key.) or a megaphone
> 
> 'bang protection? Arrivals checkpoint. Departmental sec posts.
> 
> Chameleon? Cham kit.
> 
> Ability to speak on any comms channel? Yeah at the cost of not hearing common or said channel you plan to speak on w/o a syndie key anyway.
> 
> 25 TC is absolutely overpriced for this.

Since it's obvious you don't want to be productive about this and are instead speaking in circles, please refer to the latest announcement posted by Xyel - on 4/16/21 specifically - if you have any further questions, as I will no longer be available to you for comment. That is all.

**Comment by Gandalf2k15** [comment]
Can people not read announcements? Keep this shit to discord @FlamingLily

**Comment by Gandalf2k15** [comment]
I'd say 4TC for the headset and 5 for the ID
